modules/oe_webtools_etrans/src/Plugin/Block/UnifiedEtransBlock.php

```diff
@@ -101,10 +101,14 @@ public function build(): array {
     $placeholder_id = Html::getUniqueId('etrans-widget');
     // On node route, we take the node original language,
     // which is not necessarily the default language of the website.
-    $translation_from = $this->getRouteEntityLangcode(TRUE) ?: $this->languageManager->getDefaultLanguage()->getId();
-    $translation_to_language = $this->languageManager->getCurrentLanguage();
+    $current_language = $this->languageManager->getCurrentLanguage();
+    $langcode_from = $this->getRouteEntityLangcode(TRUE) ?: $this->languageManager->getDefaultLanguage()->getId();
+    $langcode_to = $current_language->getId();
 
-    $json = $this->preparesWtEtransJson($translation_from, $placeholder_id);
+    $langcode_from = $this->mapLangcodeToIso($langcode_from);
+    $langcode_to = $this->mapLangcodeToIso($langcode_to);
+
+    $json = $this->preparesWtEtransJson($langcode_from, $placeholder_id);
```

Comment:
I think we should use different variable names for the ISO langcodes, I think it would be more clear.
```suggestion
    $iso_langcode_from = $this->mapLangcodeToIso($langcode_from);
    $iso_langcode_to = $this->mapLangcodeToIso($langcode_to);

    $json = $this->preparesWtEtransJson($iso_langcode_from, $placeholder_id);
```

Reply:
Of course the idea would be to use two distinct variable names:
- The Drupal langcode is `$langcode_from`.
- The ISO langcode is `$iso_langcode_from`.

So yes, we need to apply the rename further down, as you did (I did not want to go crazy with this suggestion).
But before the `->mapLangcodeToIso()` we want to keep the old name.

Reply:
@donquixote, let's not block this PR for such details.
If really that doesn't suit you, feel free to commit your changes.

Reply:
I am happy to do that! Thanks for the green light.